src/js/editor/editor.js

```diff
@@ -769,6 +772,10 @@ class Editor {
       cardSection.setInitialMode(mode);
     }
   }
+
+  get hasRendered() {
+    return !!this.element;
```

Comment:
technically `element` is present before rendering occurs. seems good for a start though.